### ISSUE TITLE
任务三分类

### ISSUE THREAD

**Issue by Pythonisfun** [yuanqing811/ISIC2018#1]
你们在任务三分类，除此之外还有哪些尝试和改进？谢谢。

**Comment by yuanqing811** [comment]
> What are the other attempts and improvements that you have made in Task Three? Thank you.

We tried mainly the following approaches:

- Different backbones: inception, vgg, resnet, densenet
- Different top layers: tried up to 2 fully connected top layers of different sizes
- Different data augmentations: both from keras and imgaug library. Finally, we limited to only those augmentations that help.
- Regularization: L1, L2, L1 + L2, dropout

Things to try:

- Data normalization
- Other loss functions (focal loss etc)

**Comment by tavildar** [comment]
Closing the issue. Please see the updated results for Task 3 that take into account the lesion ID split.
